packages/dashboard/src/components/Widget/Widget.tsx

```diff
@@ -71,7 +72,24 @@ const Widget = ({
 
   const deleteWidget = () => {
     if (!widgetConfig) return
+    // if last widget left in row remove whole row on delete
+    if (widgetConfig.colIdx === 0) {
```

Comment:
what if it's the last column but it's at index 2?

Reply:
@joshlacey  thats not issue. it will delete only last(one) widget.